app/controllers/users_controller.rb

```diff
@@ -30,4 +30,18 @@ def sort_update
 
     redirect_to :sort_users, success: 'Guider order has been updated'
   end
+
+  def deactivate
+    user = User.find(params[:user_id])
```

Comment:
Maybe extracting lines 35-37 into something like this might remove some of that duplication and tidy this up a teeny bit?

```ruby
def toggle_activation
  user = User.find(params[:user_id])
  user.toggle!(:active)

  GenerateBookableSlotsForUserJob.perform_later(user)
end
```